Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -113,17 +102,22 @@
     return graphicsLayer;
 }
 
-void UnifiedPDFPlugin::udpateLayerHierarchy()
+void UnifiedPDFPlugin::updateLayerHierarchy()
 {
-    m_rootLayer = createGraphicsLayer("UnifiedPDFPlugin root"_s, GraphicsLayer::Type::Normal);
-    m_rootLayer->setAnchorPoint({ });
+    if (!m_rootLayer) {
+        m_rootLayer = createGraphicsLayer("UnifiedPDFPlugin root"_s, GraphicsLayer::Type::Normal);
+        m_rootLayer->setAnchorPoint({ });
+    }
 
-    m_contentsLayer = createGraphicsLayer("UnifiedPDFPlugin contents"_s, GraphicsLayer::Type::Normal);
-    m_contentsLayer->setAnchorPoint({ });
-    m_contentsLayer->setSize(size());
-    m_contentsLayer->setDrawsContent(true);
+    if (!m_contentsLayer) {
+        m_contentsLayer = createGraphicsLayer("UnifiedPDFPlugin contents"_s, GraphicsLayer::Type::Normal);
+        m_contentsLayer->setAnchorPoint({ });
```

Comment:
Per our new smart pointer idioms, you want to be calling these methods on a local copy of the RefPtr, not directly on the data member. We must assume that data members can be nulled out at any point and thus this isn't guaranteed safe.

Should be something like:
```
auto contentsLayer = createGraphicsLayer("UnifiedPDFPlugin contents"_s, GraphicsLayer::Type::Normal);
m_contentsLayer = contentsLayer.copyRef();
contentsLayer->setAnchorPoint({ });
...
```